src/File/Thumbnailer/VideoAwareThumbnailer.php

```
@@ -0,0 +1,404 @@
+                    unlink($sourcePathWithExtension);
+                    $this->sourceFile = $originalSourceFile;
+                    throw $e;
+                }
+            } else {
+                error_log('VideoAwareThumbnailer: Failed to copy file with extension');
+            }
+        }
+
+        error_log('VideoAwareThumbnailer: Calling parent ImageMagick directly');
+        return parent::create($strategy, $constraint, $options);
+    }
```

PHPMD (2.15.0)

The method create() has a Cyclomatic Complexity of 11. The configured cyclomatic complexity threshold is 10. (Code Size Rules)

(CyclomaticComplexity)